src/spz/spz/views.py

```diff
@@ -421,23 +421,21 @@ def notifications():
 
 @login_required
 @templated('internal/export.html')
-def export(type, id, format='xlsx'):
-    if type == 'course':
-        form = forms.ExportCourseForm()
-    elif type == 'language':
-        form = forms.ExportLanguageForm()
-    else:
-        abort(404)
+def export(type, id):
```

Comment:
Trying to export the entire "spanish" language does not show the custom `Spanisch Vorlage (.zip)` formatter

Reply:
Currently the language of the authenticated user determines, which formatters to show. Since there is no need for anybody, to export languages in another users format, I would keep things as they are for now.

Reply:
Ah, I see 👍